back/app/models/notifications/project_phase_started.rb

```diff
@@ -72,6 +72,7 @@ class ProjectPhaseStarted < Notification
 
     def self.make_notifications_on(activity)
       phase = activity.item
+      return [] unless phase.project.published?
```

Comment:
Could have used `unless phase.project.hidden?` but decided that it didn't make sense for notifications to be made unless the project was live

Reply:
Looked at this and I think adding this to inverse scope would restrict it too much so going to leave it as it is